src/CreateCategoryModal.tsx

```diff
@@ -0,0 +1,68 @@
+import {
+    Box,
+    Text,
+    NumberInput,
+    Modal,
+    ModalOverlay,
+    ModalContent,
+    ModalHeader,
+    ModalFooter,
+    ModalBody,
+    ModalCloseButton,
+    useDisclosure,
+    Button,
+    Input,
+    Stack,
+    NumberInputField,
+    NumberInputStepper,
+    NumberIncrementStepper,
+    NumberDecrementStepper
+  } from "@chakra-ui/react"
+import * as React from "react"
+
+interface CategoryModal {
+  goals: string[];
+}
+/*
+void()=> print(onCloseFunction:Function) {
+  console.log("test");
+  onCloseFunction();
+}
+*/
+export const CreateCategoryModal = (props: CategoryModal) => {
+    const{isOpen, onOpen, onClose} = useDisclosure()
+    
+    
+
+    return (
+    <Box p={4}>
+      <Button onClick={onOpen}>Open Modal</Button>
```

Comment:
Hmm,  we already have a button for this on the sidebar in that PR we just merged in. We should probably remove this now?

Reply:
Yeah it would be replaced.